com.unity.postprocessing/PostProcessing/Runtime/PostProcessManager.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using UnityEngine.Assertions;
+using UnityEngine.Pool;
```

Comment:
Needs a version bump in the package info as it's currently set to 2019.4.19f. AFAIK that package was introduced in 2021? That's quite a bump for a bug fix :(

Reply:
I can workaround the use of `UnityEngine.Pool` to avoid the version bump.

Reply:
Fixed in 8349088